### ISSUE TITLE
Applying any update to Essences fails [Steam save] [GUI]

### ISSUE THREAD

**Issue by avantearpeggio** [fnr1r/kadishutu#5]
```
Traceback (most recent call last):
  File "C:\Users\$USER\pipx\venvs\kadishutu\Lib\site-packages\kadishutu\gui\mainwindow.py", line 297, in on_apply
    widget.on_apply_changes()
  File "C:\Users\$USER\pipx\venvs\kadishutu\Lib\site-packages\kadishutu\gui\game_save\essences.py", line 53, in on_apply_changes
    meta_box.setattr_if_modified(essence, "metadata")
  File "C:\Users\$USER\pipx\venvs\kadishutu\Lib\site-packages\kadishutu\gui\shared.py", line 54, in setattr_if_modified
    obj.__setattr__(attr, self.get_value())
  File "C:\Users\$USER\pipx\venvs\kadishutu\Lib\site-packages\kadishutu\core\shared\editors.py", line 236, in __set__
    self.write(value)
  File "C:\Users\$USER\pipx\venvs\kadishutu\Lib\site-packages\kadishutu\core\shared\editors.py", line 447, in write
    super().write(v.value)
                  ^^^^^^^
AttributeError: 'int' object has no attribute 'value'
```

This happens on any update, including when you make no changes to values.

To repro:
1. Open a steam sav (or a decrypted switch save?)
2. Open the essence menu
3. Choose "Apply" without making changes
4. See the `attr` setting errors.

**Comment by fnr1r** [comment]
Fixed in [1fb1706](https://github.com/fnr1r/kadishutu/commit/1fb170612c5a4ee4b52684ab6daaa3eae3220a9a). Thanks for reporting this.

**Comment by fnr1r** [comment]
`alpha-be6b376` is published. I'm closing this issue.
